lib/pf/dhcp/processor.pm

```diff
@@ -274,7 +296,7 @@ sub parse_dhcp_request {
 
     # We check if we are running without dhcpd
     # This means we don't see ACK so we need to act on requests
-    if((!$self->{running_w_dhcpd} && !isenabled($Config{network}{force_listener_update_on_ack})) && (defined($client_ip) && defined($client_mac))){
+    if( (defined($client_ip) && defined($client_mac)) && (!$self->pf_is_dhcp($client_ip) && !isenabled($Config{network}{force_listener_update_on_ack})) ){
```

Comment:
Don't we have to remove !isenabled($Config{network}{force_listener_update_on_ack} condition ?, if packetfence is not the dhcp for this client then we update the iplog since we are not sure that we will receive the ACK packet.